database/factories/DimensionFactory.php

```diff
@@ -17,7 +17,7 @@ public function definition(): array
             'created_at' => Carbon::now(),
             'updated_at' => Carbon::now(),
 
-            'topic_id' => Topic::factory(),
+            // 'topic_id' => Topic::factory(),
```

Comment:
Likewise here - not that we're ever going to use these factories or seeders again though...

Reply:
Yes.